src/QuantumToolbox.jl

```diff
@@ -32,6 +32,7 @@ import OrdinaryDiffEq: OrdinaryDiffEqAlgorithm
 import Pkg
 import Random
 import SpecialFunctions: loggamma
+import StaticArraysCore: SVector, MVector
```

Comment:
Since we mention about `SVector` a lot in documentation and also some warning messages.
I think it would be better to reexport `SVector`:
```julia
@reexport import StaticArraysCore: SVector # put this together with other re-export packages
import StaticArraysCore: MVector # put this here
```